src/vsr/client_replies.zig

```diff
@@ -176,17 +194,23 @@ pub fn ClientRepliesType(comptime Storage: type) type {
                 client_replies.write_reply_next();
             }
 
-            if (!message.header.valid_checksum()) {
+            if (!message.header.valid_checksum() or
+                !message.header.valid_checksum_body(message.body()))
+            {
                 log.warn("{}: read_reply: corrupt header (client={} reply={})", .{
```

Comment:
```suggestion
                log.warn("{}: read_reply: corrupt reply (client={} reply={})", .{
```

Reply:
Well, not  sure what's the best phrasing,but its not only the header that can be an issue